test.js

```diff
@@ -1,136 +1,129 @@
 'use strict'
 
-var test = require('tape')
-var hapi = require('hapi')
-var boom = require('boom')
-var semver = require('semver')
-var proxyquire = require('proxyquire')
+const test = require('tape')
+const hapi = require('hapi')
+const boom = require('boom')
+const semver = require('semver')
+const proxyquire = require('proxyquire')
 
-var hapiVersion = semver.major(require('hapi/package.json').version)
+test('hapi version 17', t => {
+  t.plan(1)
+
+  const hapiVersion = semver.major(require('hapi/package.json').version)
+  t.ok(hapiVersion >= 17)
+})
 
-test('options', function (t) {
+test('options', t => {
   t.plan(2)
 
-  var server = Server()
-  var plugin = proxyquire('./', {
+  const server = Server()
+  const plugin = proxyquire('./', {
     raven: {
-      config: function testConfig (dsn, options) {
+      config (dsn, options) {
         t.equal(dsn, 'dsn')
-        t.deepEqual(options, {foo: 'bar'})
+        t.deepEqual(options, { foo: 'bar' })
       }
     }
   })
 
   register(server, plugin, {
     dsn: 'dsn',
-    client: {foo: 'bar'}
+    client: { foo: 'bar' }
   })
 })
 
-test('request-error', function (t) {
+test('request-error', async t => {
```

Comment:
don't throw `async` before a function just because. this is not an async function.

Reply:
well it is async.